### PR TITLE
Image & Input Component

### PR DESCRIPTION
**Pull request by manojncsoni007**
Screenshot (176)

### CODE REVIEW

**Review by dcs-soni — APPROVED** [review]
Looks good to me 👍 Can remove unwanted spaces from the files.

Also, try to add a preview in the PRs. It'll be helpful for the reviewers.

File: Components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Aditi0505) [inline Components/image/image.html:8]: Please add a meaningful title.

File: Components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Aditi0505) [inline Components/input/input.html:8]: Same as above.

File: Components/input/input.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <label for="number" class="label text-red">Enter Name *</label>
+    <input type="number" id="number" class="input error-outline" placeholder="Enter Number">
+
+
+
+
+
+
+
+
```
Comment (Aditi0505) [inline Components/input/input.html:46]: Please remove unnecessary spacing.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. rest looks good to me.

**Pull request merged**